torch/onnx/_internal/exporter.py

```diff
@@ -545,6 +545,10 @@ def _export_fx_to_onnx(
             # ONNX does not support None inputs. During graph building, all None inputs
             # are removed. Here we register this step to input adapter.
             self.adapt_input(fx_exporter.RemoveNoneInputStep, fx_module_args, {})
+            # TODO(titaiwang): temp workaround for https://github.com/pytorch/pytorch/issues/99534
+            # Dynamo doesn't support constant inputs, but could change. We can remove this
```

Comment:
I thought from the issue conversation, builtin type input is never going to be supported by dynamo?

Reply:
I just wanted to keep a NOTE though. I will rephrase.

Reply:
Done